<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IUGF0FI" failures="1" tests="3" name="Default test(failed)" time="0.145" errors="0" timestamp="2022-09-05T11:13:19 IST">
  <testcase classname="TestLayer.dimenti" name="m3" time="0.018"/>
  <testcase classname="TestLayer.dimenti" name="m2" time="0.002"/>
  <testcase classname="TestLayer.dimenti" name="m1" time="0.019">
    <failure type="java.lang.AssertionError" message="expected [12] but found [11]">
      <![CDATA[java.lang.AssertionError: expected [12] but found [11]
at TestLayer.dimenti.m1(dimenti.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- m1 -->
</testsuite> <!-- Default test(failed) -->
